plugins/python-build/share/python-build/pypy2.7-portable-5.9.0

```diff
@@ -0,0 +1,13 @@
+case "$(pypy_architecture 2>/dev/null || true)" in
+"linux64" )
+  install_package "pypy-5.9-linux_x86_64-portable" "https://bitbucket.org/squeaky/portable-pypy/downloads/pypy-5.9-1-linux_x86_64-portable.tar.bz2#8d39eb98df3adf7882a7f3551f47b8c7cff47a0e20d6aabc57bb592f155c2616" "pypy" verify_py27 ensurepip
```

Comment:
Looks like this link has a typo and so is giving 404. (There is a spurious "-1" in the filename – it should be https://bitbucket.org/squeaky/portable-pypy/downloads/pypy-5.9-linux_x86_64-portable.tar.bz2, not https://bitbucket.org/squeaky/portable-pypy/downloads/pypy-5.9-1-linux_x86_64-portable.tar.bz2.) Submitted a PR to fix this in #1010.